pkg/virt-handler/node-labeller/cpu_plugin_test.go

```diff
@@ -74,20 +82,15 @@ var _ = Describe("Node-labeller config", func() {
 		err = nlController.loadHostSupportedFeatures()
 		Expect(err).ToNot(HaveOccurred())
 
-		err = nlController.loadHostCapabilities()
-		Expect(err).ToNot(HaveOccurred())
-
 		cpuModels := nlController.getSupportedCpuModels(nlController.clusterConfig.GetObsoleteCPUModels())
 		cpuFeatures := nlController.getSupportedCpuFeatures()
 
 		Expect(cpuModels).To(HaveLen(4), "number of models must match")
-
 		Expect(cpuFeatures).To(HaveLen(4), "number of features must match")
-		counter := nlController.GetTSCCounter()
-		Expect(err).ToNot(HaveOccurred())
-		Expect(counter).ToNot(BeNil())
-		Expect(counter.Frequency).To(BeNumerically("==", 4008012000))
 
+		Expect(nlController.cpuCounter).ToNot(BeNil())
+		Expect(nlController.cpuCounter.Name).To(Equal("tsc"))
+		Expect(nlController.cpuCounter.Frequency).To(BeNumerically("==", 4008012000))
```

Comment:
Since there is a better test under `node_labeller_test.go`, I'm not sure we need to keep this one as well.
It asserts the internal state of `nlController` (which can break) while the other test tests its observed behavior.

Reply:
That's fair, I kept it since the original test had it but now that we have a better one I agree that it's unnecessary.